src/wallet/crypter.h

```diff
@@ -58,11 +59,11 @@ class CMasterKey
 
     CMasterKey()
     {
-        // 25000 rounds is just under 0.1 seconds on a 1.86 GHz Pentium M
-        // ie slightly lower than the lowest hardware we need bother supporting
-        nDeriveIterations = 25000;
         nDerivationMethod = 0;
-        vchOtherDerivationParameters = std::vector<unsigned char>(0);
+        nDeriveIterations = crypto_pwhash_OPSLIMIT_INTERACTIVE;
+        size_t memlimit = crypto_pwhash_MEMLIMIT_INTERACTIVE;
+        vchOtherDerivationParameters = std::vector<unsigned char>(sizeof(size_t));
+        ::memcpy(vchOtherDerivationParameters.data(), &memlimit, sizeof(memlimit));
```

Comment:
Yes, you should serialize with compact size. Even the endianness here is implementation specific.